On Void, use the non-forking option -D when starting vncsession

--- a/unix/vncserver/vncsession-start.in	2023-02-28 10:11:20.000000000 -0500
+++ b/unix/vncserver/vncsession-start.in	2023-07-30 17:53:09.330000000 -0400
@@ -40,4 +40,4 @@
 	exit 1
 fi
 
-exec "@CMAKE_INSTALL_FULL_SBINDIR@/vncsession" "${USER}" "${DISPLAY}"
+exec "@CMAKE_INSTALL_FULL_SBINDIR@/vncsession" -D "${USER}" "${DISPLAY}"
